string type
ScenarioPose start_pose
ScenarioPose end_pose
float64 speed